frontend/routes/teamspaces/teamspaces.component.tsx

```diff
@@ -467,19 +468,13 @@ export class Teamspaces extends React.PureComponent<IProps, IState> {
 						} }
 					/>
 				</Head>
-				<List>
-					{
-						isPending ? (
-							<LoaderContainer>
-								<Loader content="Loading teamspaces..." />
-							</LoaderContainer>
-						) : (
-							<SimpleBar>
-								{this.renderTeamspaces(this.props.teamspaces)}
-							</SimpleBar>
-						)
-					}
-				</List>
+				<BodyWrapper>
```

Comment:
This change (I think) has made the loading screen disappear.
(left is staging right is this branch)
![ham](https://user-images.githubusercontent.com/11945337/65870193-87c3a180-e373-11e9-8247-a941b19e4275.gif)